src/Forms/MultiLineTextInput.tsx

```diff
@@ -0,0 +1,112 @@
+import React, {
+  ReactElement,
+  FunctionComponent,
+  ChangeEventHandler,
+  useContext,
+  Ref,
+} from 'react';
+import styled, { ThemeContext } from 'styled-components';
+import { fromTheme } from '../Theme';
+import InputLabel, { InputLabelPosition, POSITION } from './InputLabel';
+import ValidationErrorMessage from './ValidationErrorMessage';
+
+export interface MultiLineTextInputProps {
+  /** The id of the label tied to this textarea */
+  id: string;
+  /** The body of the error message, if applicable */
+  errorMessage?: string;
+  /** The value of the multi-line text area */
+  value?: string;
+  /** Function to call on change event */
+  onChange: ChangeEventHandler;
+  /** Marks the field as required */
+  isRequired?: boolean;
+  /** The label to display beside this multi-line input field(if any) */
+  label: string;
+  /** Whether to show or hide the label */
+  isLabelVisible?: boolean;
+  /** Disables the field entirely and prevents the onChange handler from being called */
+  isDisabled?: boolean;
+  /** Specify the position of the input label */
+  labelPosition?: InputLabelPosition;
+  /** Specifies the ref of the text input */
+  forwardRef?: Ref<HTMLTextAreaElement>;
+  /** The value to set for the `name` prop of the underlying `textarea` element */
+  name: string;
+  /** Provides a placeholder value for the `textarea` */
+  placeholder: string;
+}
+
+export const StyledTextArea = styled.textarea<MultiLineTextInputProps>`
```

Comment:
Do you know if it is expected that the label and text area need to be aligned vertically like they are for the `TextInput` component? Maybe this is something to ask Vittorio? Although, I think our currently existing use case for the text area component is the notes modal in Course Planner, and that doesn't include a label.

Reply:
I also wonder if we should have a visible scrollbar when the text takes up more room than the size of text area box. Maybe that's another question for Vittorio?

Reply:
@Vittorio2015 may have some thoughts here, but I thought this was the default behavior for textarea elements

Reply:
> Do you know if it is expected that the label and text area need to be aligned vertically like they are for the `TextInput` component? Maybe this is something to ask Vittorio? Although, I think our currently existing use case for the text area component is the notes modal in Course Planner, and that doesn't include a label.

This might be something to chat about in the 9:30 standup?

> I also wonder if we should have a visible scrollbar when the text takes up more room than the size of text area box. Maybe that's another question for Vittorio?

@Vittorio2015 may have some thoughts here, but I thought this was the default behavior for textarea elements